lib/stub_ui/lib/src/ui/window.dart

```diff
@@ -35,6 +35,71 @@ typedef PlatformMessageResponseCallback = void Function(ByteData data);
 typedef PlatformMessageCallback = void Function(
     String name, ByteData data, PlatformMessageResponseCallback callback);
 
+/// Time-related performance metrics of a frame.
+class FrameTiming {
+  const FrameTiming._(this.buildStart, this.buildFinish, this.rasterStart, this.rasterFinish);
```

Comment:
why no public constructor?

Reply:
Nevermind, I made it public for unit tests. Documentations are also added.

Reply:
WindowPadding is a bit of a special case because we want to drive people towards EdgeInsets instead, that's why it doesn't have a constructor